core/modules/replica/FindAllRequest.h

```diff
@@ -128,56 +166,41 @@ class FindAllRequest
                    bool keepTracking,
                    std::shared_ptr<Messenger> const& messenger);
 
-    /**
-      * @see Request::startImpl()
-      */
-    void startImpl(util::Lock const& lock) final;
-
     /**
      * Start the timer before attempting the previously failed
      * or successful (if a status check is needed) step.
      *
-     * @param lock - a lock on a mutex must be acquired before calling this method
+     * @param lock
+     *   a lock on a mutex must be acquired before calling this method
      */
-    void wait(util::Lock const& lock);
+    void _wait(util::Lock const& lock);
 
     /**
      * Callback handler for the asynchronous operation
      *
      * @param ec - error code to be checked
      */
-    void awaken(boost::system::error_code const& ec);
+    void _awaken(boost::system::error_code const& ec);
 
     /**
      * Send the serialized content of the buffer to a worker
      *
```

Comment:
I don't think these blank line really help readability. The '@' makes a pretty distinctive visual break by itself.

Reply:
noted